BeeJet.Bot/Commands/Handlers/GameManagement/GameButtonsPressedHandler.cs

```diff
@@ -4,18 +4,19 @@
 
 namespace BeeJet.Bot.Commands.Handlers.GameManagement
 {
-    public class GameButtonsPressedHandler : IButtonPressedHandler
+    public class GameButtonsPressedHandler : ButtonPressedHandler
     {
         [ButtonPressedHandler(AddGameCommandHandler.JointButtonId)]
-        public async Task JoinGamePressed(SocketMessageComponent component)
+        public async Task JoinGamePressed()
         {
-            if (TryGetGameName(component.Message, out string gameName))
+            if (TryGetGameName(Context.Message, out string gameName))
             {
-                await JoinGameAsync(component.Message, component.User, gameName);
+                await JoinGameAsync(Context.Message, Context.User, gameName);
```

Comment:
Je gebruikt in het bestand Context.User en soms geef je IUser mee. Is dat logisch?
Kunnen ze verschillen?

Reply:
Zijn het zelfde, lijkt mij dat het nu beter te unittesten is? 
Nu heb je niet de hele context nodig om bv JoinGameAsync te testen alleen een message en een user.

Reply:
Die suggestie krijg je niet vanuit de code (dat het hetzelfde is)
Voor unit testen is het ook niet per se makkelijker.
Want ik kan nu Context.User substituten en die werkt voor beide.
Anders moet ik los de Context.User substituten en User los substituten (afhankelijk of ik hem los mee moet geven).

Het kan allebei. Betekend voor mijn test soms meerdere keren substituten voor hetzelfde.

Reply:
Dan pas ik het nog even aan